### PR TITLE
fixed tablet view of event-types

### PR DESCRIPTION
**Pull request by PeerRich**
before:

https://user-images.githubusercontent.com/8019099/157542419-56ad9644-4216-40ec-a69f-2e98ea118058.mov

after:

https://user-images.githubusercontent.com/8019099/157542436-f4a64e5f-cf96-4292-ab71-b3562c8df594.mov

### CODE REVIEW

File: apps/web/components/pages/eventtypes/CustomInputTypeForm.tsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ const CustomInputTypeForm: FC<Props> = (props) => {
             type="text"
             id="label"
             required
-            className="focus:border-primary-500 focus:ring-primary-500 block w-full rounded-sm border-gray-300 shadow-sm sm:text-sm"
```
Comment (PeerRich) [inline apps/web/components/pages/eventtypes/CustomInputTypeForm.tsx:76]: no idea why we have introduced bigger text in input boxes on mobile. just looks odd. removed it and gave everything "text-sm"

File: apps/web/components/team/TeamSettings.tsx [hunk 2]
```diff
@@ -89,7 +89,7 @@ export default function TeamSettings(props: Props) {
                       name="" // typescript requires name but we don't want component to render name label
                       id="team-url"
                       addOnLeading={
-                        <span className="inline-flex items-center rounded-l-sm border border-r-0 border-gray-300 bg-gray-50 px-3 text-gray-500 sm:text-sm">
```
Comment (PeerRich) [inline apps/web/components/team/TeamSettings.tsx:92]: same here

File: apps/web/pages/auth/signup.tsx [hunk 3]
```diff
@@ -89,7 +89,7 @@ export default function Signup({ email }: Props) {
               <div className="space-y-2">
                 <TextField
                   addOnLeading={
-                    <span className="inline-flex items-center rounded-l-sm border border-r-0 border-gray-300 bg-gray-50 px-3 text-gray-500 sm:text-sm">
```
Comment (PeerRich) [inline apps/web/pages/auth/signup.tsx:92]: same here

File: apps/web/pages/event-types/[type].tsx [hunk 4]
```diff
@@ -648,7 +648,6 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
   return (
     <div>
       <Shell
-        centered
```
Comment (PeerRich) [inline apps/web/pages/event-types/[type].tsx:651]: for consistency, I've uncentered event-types detail view.

we may as well get rid of the "centered" prop for Shell.

File: apps/web/pages/event-types/[type].tsx [hunk 5]
```diff
@@ -679,8 +678,8 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
         }
         subtitle={eventType.description || ""}>
         <ClientSuspense fallback={<Loader />}>
-          <div className="mx-auto block sm:flex md:max-w-5xl">
-            <div className="w-full ltr:mr-2 rtl:ml-2 sm:w-9/12">
+          <div className="flex flex-col-reverse lg:flex-row">
```
Comment (PeerRich) [inline apps/web/pages/event-types/[type].tsx:681]: flex-col-reverse so the action buttons are at the top

File: apps/web/pages/event-types/[type].tsx [hunk 6]
```diff
@@ -1484,8 +1484,8 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
                   {t("copy_link")}
                 </button>
                 <Dialog>
-                  <DialogTrigger className="text-md flex items-center rounded-sm px-2 py-1 text-sm font-medium text-neutral-700 hover:bg-gray-200 hover:text-gray-900">
-                    <TrashIcon className="h-4 w-4 text-neutral-500 ltr:mr-2 rtl:ml-2" />
```
Comment (PeerRich) [inline apps/web/pages/event-types/[type].tsx:1488]: i feel like i wanted to highlight the delete button so I made it red

**Review by zomars — APPROVED** [review]
This is so much easier to review. Thank you @PeerRich

**Comment by PeerRich** [comment]
> This is so much easier to review. Thank you @PeerRich

I love the idea of self-reviews. Let's make this mandatory-ish

**Review by alannnc — APPROVED** [review]
Tested and looked good!

**Pull request merged**